page-about.php

```diff
@@ -0,0 +1,100 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+    $meta = get_post_meta($post->ID);
+?>
+  <!-- main posts loop -->
+  <section id="page" class="container margin-bottom-large">
+    <div class="row margin-bottom-basic">
+      <div class="col col24">
+        <h4><?php the_title(); ?></h4>
+      </div>
+    </div>
+    <div class="row margin-bottom-mid">
+      <div class="col col10">
+        <?php the_content(); ?>
+      </div>
+      <div class="col col2"></div>
+      <div class="col col10">
+        <?php if (!empty($meta['_cmb_page_extra'])) {
+          echo apply_filters('the_content', $meta['_cmb_page_extra'][0]);
+        } ?>
+      </div>
+    </div>
+    <div class="row margin-bottom-basic">
+      <div class="col col24">
+        <h5>Editorial Team</h5>
+      </div>
+    </div>
+    <div class="row margin-bottom-mid">
+<?php
+    $people = get_posts(array('post_type' => 'person', 'posts_per_page' => -1, 'order' => 'ASC'));
```

Comment:
why not use a WP_Query and avoid stuff like:
`<?php echo wp_get_attachment_image(get_post_thumbnail_id($person->ID), 'col4'); ?>`
`<?php echo apply_filters('the_content', $person->post_content); ?>`
??